src/tests/sqlsmith/src/sql_gen/query.rs

```diff
@@ -133,7 +133,7 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         let from = None;
         let cte = Cte {
             alias: alias.clone(),
-            query,
+            query: Some(query),
```

Comment:
How hard it is to support changelog cte in sqlsmith as well?

Reply:
will do it in next pr